samples/net/mqtt_publisher/prj.conf

```diff
@@ -31,3 +31,6 @@ CONFIG_MAIN_STACK_SIZE=2048
 CONFIG_NET_BUF_DATA_SIZE=256
```

Comment:
Commit subject is ok, but lets tweak the commit message a bit so that it is more useful. Can you change it to
```
Add missing random options to prj.conf file.

Fixes #31414

Signed-off-by: Vitaliy Gribko <vitaliy69@gmail.com>
```

Reply:
> Can you change

Updated